web/packages/design/src/theme/darkTheme.ts

```diff
@@ -0,0 +1,196 @@
+limitations under the License.
+*/
+
+import { fonts } from './fonts';
+import { getContrastRatio } from './utils/colorManipulator';
+import { lightBlue, blueGrey, yellow } from './palette';
+import typography, { fontSizes, fontWeights } from './typography';
+import { sharedStyles } from './sharedStyles';
+
+const space = [0, 4, 8, 16, 24, 32, 40, 48, 56, 64, 72, 80];
+const contrastThreshold = 3;
+
+const colors = {
+  /*
+  Colors in `levels` are used to reflect the perceived depth of elements in the UI.
+  The further back an element is, the more "sunken" it is, and the more forwards it is, the more "elevated" it is (think CSS z-index).
+
+  A `sunken` color would be used to represent something like the background of the app.
+  While `surface` would be the color of the primary surface where most content is located (such as tables).
+  Any colors more "elevated" than that would be used for things such as popovers, menus, and dialogs.
+
+  For more information on this concept: https://m3.material.io/styles/elevation/applying-elevation
+ */
+  levels: {
+    deep: '#000000',
+
+    sunken: '#0C143D',
+
+    surface: '#222C59',
+
+    elevated: '#344179',
+
+    popout: '#4A5688',
+  },
+
+  // Spot backgrounds are used as highlights, for example
+  // to indicate a hover or active state for an item in a menu.
+  spotBackground: [
+    'rgba(255,255,255,0.07)',
+    'rgba(255,255,255,0.13)',
+    'rgba(255,255,255,0.18)',
+  ],
+
+  brand: '#9F85FF',
+
+  text: {
+    // The most important text.
+    primary: '#FFFFFF',
+    // Secondary text.
+    secondary: 'rgba(255, 255, 255, 0.56)',
+    // Placeholder text for forms.
+    placeholder: 'rgba(255, 255, 255, 0.24)',
+    // Disabled text have even lower visual prominence.
+    disabled: 'rgba(255, 255, 255, 0.18)',
+    // For text on  a background that is on a color opposite to the theme. For dark theme,
+    // this would mean text that is on a light background.
+    primaryInverse: '#000000',
+  },
+
+  buttons: {
+    text: 'rgba(255,255,255,0.87)',
+    textDisabled: 'rgba(255, 255, 255, 0.3)',
+    bgDisabled: 'rgba(255, 255, 255, 0.12)',
+
+    primary: {
+      text: '#000000',
+      default: '#9F85FF',
+      hover: '#B29DFF',
+      active: '#C5B6FF',
+    },
+
+    secondary: {
+      default: 'rgba(255,255,255,0.07)',
+      hover: 'rgba(255,255,255,0.13)',
+      active: 'rgba(255,255,255,0.18)',
+    },
+
+    border: {
+      default: 'rgba(255,255,255,0)',
+      hover: 'rgba(255, 255, 255, 0.07)',
+      active: 'rgba(255, 255, 255, 0.13)',
+      border: 'rgba(255, 255, 255, 0.36)',
+    },
+
+    warning: {
+      text: '#000000',
+      default: '#FF6257',
+      hover: '#FF8179',
+      active: '#FFA19A',
+    },
+
+    trashButton: {
+      default: 'rgba(255, 255, 255, 0.07)',
+      hover: 'rgba(255, 255, 255, 0.13)',
+    },
+
+    link: {
+      default: '#009EFF',
+      hover: '#33B1FF',
+      active: '#66C5FF',
+    },
```

Comment:
Some of the new colors, such as `buttons.link`, are missing from Connect's theme. Because of that, some Connect components that use shared components crash. For example, check the story of `DocumentGateway`.

Are there any other new colors other than `buttons.link`?

Reply:
There were a few new colours, I added most of them to the teleterm theme (by copying them from Teleport dark theme) but must have forgotten a couple (such as the `button.link`).